client/commands.h

```diff
@@ -22,6 +22,14 @@ extern double xdagGetHashRate(void);
 
 #define XFER_MAX_IN				11
 
+typedef int (*xdag_com_func_t)(char*, FILE *);
+
+typedef struct {
+    char *name;                  /* User printable name of the function. */
+    xdag_com_func_t func;        /* Function to call to do the job.      */
+    char *doc;                   /* Documentation for this function.     */
+} XDAG_COMMAND;
```

Comment:
This type used only command.c. Move declaration there.

Reply:
ok